test/application/widgets/theme/custom_button_test.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:memo/application/constants/images.dart' as images;
+import 'package:memo/application/widgets/theme/custom_button.dart';
+
+import '../../../utils/widget_pump.dart';
+
+void main() {
```

Comment:
Most of these tests are copy-pasted. Can't we simply test these buttons within a common logic?

Reply:
Added auxiliary functions, let me know what do you think